resources/frontend/inspect_element.md

```diff
@@ -0,0 +1,28 @@
+## Understanding Inspect Element
+Inspect Element is a browser tool that lets you view and edit the HTML and CSS. However, you cannot make
+permanent changes. **Edits made will be deleted once you exit out. 
+
+### How to Open Inspect Element
+
+- **Mac**
+    - Cmd + Shift + C to open the Developer Tools in Inspect Element mode 
+    
+- **PC**
+    - Ctrl + Shift + C to open the Developer Tools in Inspect Element mode 
+
+### Three Parts of Element
+
+- **DOM Panel**
+    - DOM panel is a tool used to change the page layout. You’ll have full control of HTML files and can modify them freely. It’s located in the upper part of the box under Elements. When you hover over the tags like <div> in DOM, the corresponding element will be highlighted in the viewport. You can expand or collapse the details for each element by clicking the grey triangle.
```

Comment:
For these, can you include screenshots please? They can be screenshots you take yourself or downloaded from the Internet. It will be easier to see what you're talking about.